bin/build-kittens.rb

```diff
@@ -0,0 +1,31 @@
+require 'net/http'
+require 'json'
+
+LIMIT = 365
+
+def run!
+  uri = URI('http://api.giphy.com/v1/gifs/search')
+  params = {
+    :limit => LIMIT,
+    :rating => 'g',
+    :q => 'cats kittens',
+    :api_key => 'dc6zaTOxFJmzC',
+  }
+  uri.query = URI.encode_www_form(params)
+
+  res = Net::HTTP.get_response(uri)
+  if res.is_a?(Net::HTTPSuccess)
+    response_body = JSON.parse(res.body, :symbolize_names => true)
+    new = response_body[:data].map do | item |
```

Comment:
new is a reserved word in Rails and so rubyists generally avoid naming vars `new`